src/integrations/prefect-kubernetes/README.md

```diff
@@ -24,7 +24,7 @@ Install `prefect-kubernetes` with `pip`:
  pip install prefect-kubernetes
  ```
 
-Requires an installation of Python 3.8+.
```

Comment:
does this PR remove 3.8 support? i think we'll need to keep 3.8 support for 2.x

Reply:
Just forgot to update readme as the 3.x version doesn't support 3.8.